README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
+
+Install Go Task
+
+```sh
+brew install go-task
+```
```

Comment:
Provide an example of what an error message might look like if dependencies are not installed correctly, especially for the `task install` command, to assist users with debugging.